dev-tools/smoke_test_rc.py

```diff
@@ -263,6 +265,8 @@ def parse_list(string):
                       help='The Elasticsearch Version to smoke-tests', required=True)
   parser.add_argument('--hash', '-s', dest='hash', default=None, required=True,
                       help='The sha1 short hash of the git commit to smoketest')
+  parser.add_argument('--release-hash', '-r', dest='release_hash', default=None, required=True,
```

Comment:
the hash from the elasticsearch commit is used to check, if it has the right one specified in the `/` url